homeassistant/components/assist_pipeline/strings.json

```diff
@@ -24,11 +24,11 @@
   },
   "issues": {
     "assist_in_progress_deprecated": {
-      "title": "{integration_name} assist in progress binary sensors are deprecated",
+      "title": "{integration_name} in progress binary sensors are deprecated",
       "fix_flow": {
         "step": {
           "confirm_disable_entity": {
-            "description": "The {integration_name} assist in progress binary sensor `{entity_id}` is deprecated.\n\nMigrate your configuration to use the corresponding `{assist_satellite_domain}` entity and then click SUBMIT to disable the assist in progress binary sensor and fix this issue."
+            "description": "The {integration_name} in progress binary sensor `{entity_id}` is deprecated.\n\nMigrate your configuration to use the corresponding `{assist_satellite_domain}` entity and then click SUBMIT to disable the in progress binary sensor and fix this issue."
```

Comment:
```suggestion
            "description": "The {integration_name} in progress binary sensor `{entity_id}` is deprecated.\n\nMigrate your configuration to use the corresponding `{assist_satellite_domain}` entity and then select SUBMIT to disable the in progress binary sensor and fix this issue."
```

Reply:
We have existing issues saying "click", but none saying press.
(Kitchen sink issues say "press", but that's not a real integration)

Reply:
I checked with @c0ffeeca7 and from my memory this was because `select` is a term that can be used for both desktop (where you click with the mouse) and mobile (where you tap on the screen)

Reply:
[MS Style Guide on click vs select](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)

Reply:
I see. Let's change all the relevant issue descriptions to "select" in a separate PR then?